src/main/java/com/kalessil/phpStorm/phpInspectionsEA/options/OptionsComponent.java

```diff
@@ -0,0 +1,163 @@
+    }
+
+    /**
+     * Creates a List.
+     *
+     * @param fieldTitle  The title of this field on panel.
+     * @param listItems   The default or current list items.
+     * @param listUpdater The event called after a modification on list items.
+     */
+    public void createList(final String fieldTitle, @NotNull final List<String> listItems, @NotNull final Runnable listUpdater) {
+        optionsPanel.add(new JLabel(fieldTitle), "wrap");
+        optionsPanel.add((new PrettyListControl(listItems) {
+            protected void fireContentsChanged() {
+                listUpdater.run();
+                super.fireContentsChanged();
+            }
+        }).getComponent(), "pushx, growx");
+    }
+
+    /**
+     * Create the checkboxes used as radios.
+     */
+    public class RadioComponent {
+        /**
+         * Store all Radio Elements handled by this Radio Component.
+         */
+        ArrayList<Element> elements = new ArrayList<>();
+
+        /**
+         * Store the current selected Radio.
+         */
+        @Nullable
+        private Element currentSelection;
+
+        /**
+         * @param fieldTitle     The title of the field on options panel.
+         * @param defaultValue   The default or current field value stored on inspection.
+         * @param updateConsumer The update consumer (called after item change).
+         */
+        public void createOption(final String fieldTitle, final boolean defaultValue, final Consumer<Boolean> updateConsumer) {
+            final Element createElement = new Element(fieldTitle, defaultValue, updateConsumer);
+            elements.add(createElement);
+
+            if (defaultValue) {
+                currentSelection = createElement;
+            }
+        }
+
+        /**
+         * Handle a JCheckBox component and it Update Consumer.
+         */
+        private class Element {
+            /**
+             * Stores the JCheckBox component related to the Element.
+             */
+            final JCheckBox checkbox;
+
+            /**
+             * Stores the Update Consumer related to the Element.
+             */
+            final Consumer<Boolean> updateConsumer;
+
+            /**
+             * Constructs the Element.
+             *
+             * @param fieldTitle     The title of the field on options panel.
+             * @param defaultValue   The default or current field value stored on inspection.
+             * @param updateConsumer The update consumer (called after item change).
+             */
+            Element(final String fieldTitle, final boolean defaultValue, final Consumer<Boolean> updateConsumer) {
+                this.updateConsumer = updateConsumer;
+
+                checkbox = new JCheckBox(fieldTitle, defaultValue);
+                checkbox.addItemListener((itemEvent) -> {
+                    final boolean isSelected = checkbox.isSelected();
+
+                    if (currentSelection != null && currentSelection != this) {
```

Comment:
Weeeeelll.... I don't agree. In all project code we have a mix of two styles: Yoda and regular.
But let talk about that on this topic: #286.